docs/ecosystem/ifttt/readme.md

```diff
@@ -11,71 +11,48 @@ meta:
 
-openHAB users can take advantage of IFTTT to realize even more use cases in their smart home!
+openHAB users can take advantage of IFTTT to realize even more use cases in their smart home! 
 
-## What is IFTTT?
+Besides the new kinds of integration possibilities, IFTTT is also a good start for new openHAB users as it makes the creation of rules dead simple - no need to do any scripting, no complex setup routines, it all works through a few mouse clicks!
 
-[If This Then That (IFTTT)](https://ifttt.com) is a very popular web-based service that allows users to create "applets", which can combine different web services into powerful automation rules.
+# Getting Started
 
-## Why use openHAB with IFTTT?
+## Requirements
+* [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/#openhab-cloud-connector) configured using [myopenHAB.org](https://myopenHAB.org)
+* [IFTTT](https://ifttt.com/join) account
 
-openHAB already integrates with [far more than 200 different smart home technologies](/addons). While there are some web services like Twitter available, this is not the primary integration focus of openHAB. Therefore IFTTT perfectly complements the features of openHAB: With its [hundreds of channels](https://ifttt.com/channels), there are limitless new creative options: Be notified through text-to-speech if you are receive a DM on twitter, be warned through your lights if your favorite stock price falls below a certain threshold or put your house in away-mode when you turn on the ignition of your car - you can get further inspiration by browsing through existing applets.
+## Configuration
 
-<div class="row da-thumbs">
-  <article class="span4 bloc">
-    <a href="https://ifttt.com/recipes/300800-update-my-presence-if-nest-says-i-m-back-home">
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt1.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+**1. Connect openHAB with IFTTT**
 
-  <div class="da-animate da-slideFromTop" style="display: block;">
-    <span class="iconWrapper iconLink icon-search" style="font-style: italic; margin-top: 63px"></span>
-  </div></a>
+IFTTT is available to all our users through the myopenHAB service. Activating the integration is easy. Just log in to your IFTTT account and activate the [openHAB service](https://ifttt.com/openhab). You will be forwarded to the [myopenHAB.org website](http://www.myopenhab.org/) to authorize the IFTTT service connection. 
```

Comment:
Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.